pkgs/desktops/gnome-3/3.20/core/gnome-shell/default.nix

```diff
@@ -40,6 +42,9 @@ stdenv.mkDerivation rec {
       --prefix XDG_DATA_DIRS : "${gnome_themes_standard}/share:$out/share:$XDG_ICON_DIRS" \
       --suffix XDG_DATA_DIRS : "$GSETTINGS_SCHEMAS_PATH"
 
+    wrapProgram "$out/bin/gnome-shell-extension-tool" \
+      --prefix PYTHONPATH : "${python3Packages.pygobject3}/${python3.sitePackages}:$PYTHONPATH"
```

Comment:
> If the scipts have #!/usr/bin/env python as a shebang, then I think patchshebangs should pick it up in the fixup phase if a python is found in PATH at build-time.

Indeed, so the patched shebangs will point to the python symlink in the environment, which is exactly what we want.